geps/gep-713/index.md

```diff
@@ -1,1157 +1,1082 @@
 
-“Meta” here is used in its Greek sense of “more comprehensive”
-or “transcending”, and “resource” rather than “object” because “metaresource”
-is more pronounceable than “metaobject”. Additionally, a single word is better
-than a phrase like “wrapper object” or “wrapper resource” overall, although both
-of those terms are effectively synonymous with “metaresource”.
+##### Mergeability (“how”)
 
-A "Policy Attachment" is a metaresource that affects the fields in existing objects
-(like Gateway or Routes), or influences the configuration that's generated in an
-underlying data plane.
+Metaresources specify so-called *merge strategies* that dictate how multiple instances of the metaresource affecting a same resource (or section of a resource) should be dealt with in terms of calculating and enforcing a single set of rules specified by the metaresources.
 
-"Direct Policy Attachment" is when a Policy object references a single object _only_,
-and only modifies the fields of or the configuration associated with that object.
+The merge strategies typically include strategies for dealing with conflicting and/or missing specs, such as for applying default and/or override values on the target resources.
 
-"Inherited Policy Attachment" is when a Policy object references a single object
-_and any child objects of that object_ (according to some defined hierarchy), and
-modifies fields of the child objects, or configuration associated with the child
-objects.
+### Policies and Policy Attachment
 
-In either case, a Policy may either affect an object by controlling the value
-of one of the existing _fields_ in the `spec` of an object, or it may add
-additional fields that are _not_ in the `spec` of the object.
+Often when applying a metaresource, the intent has to do with specifying *rules* that control the behavior of the targets in action. E.g., for networking resources, the augmented behavior specified by a metaresource targeting an object can relate to conditions to let traffic flow or to block traffic at particular points of the network; in a different scenario, it can describe rules to mutate the traffic while it flows through the network at the point represented by the target object.
 
-### Why use Policy Attachment at all?
+The semantics of *applying rules to a target* characterizes these particular instances of metaresources as *policies*, thus here emerging in relation to this pattern the concept of *Policy Attachment*.
 
+Policies are therefore a strict subset of metaresources which distinguish itself subtly from the whole on the fact that the augmented behavior described by policies resemble *conditional rules*, while the mechanism of targeting an object or set of objects (“attaching to objects”) is nonetheless exactly the same as in the general definition of metaresources.
```

Comment:
This may be easier to communicate with a simple table or diagram that illustrates that policies are a subset of metaresources.

Reply:
I ended up removing this from the text. I kept only the one-liner suggested by @kflynn defining policies as "an example of metaresources whose intent is to specify rules…"